Makefile

```diff
@@ -17,7 +17,14 @@ certs:
 install: certs
 	kubectl create ns $(NAMESPACE) || true
 	kubectl config set-context --current --namespace $(NAMESPACE)
-	helm install connaisseur helm --wait
+	#
+	#=============================================
+	#
+	# The installation may last up to 5 minutes.
+	#
+	#=============================================
+	#
```

Comment:
Why the large comment? Is it displayed during install, or only visible in the file source code? (not sure how `make` behaves)

Reply:
The former, it is displayed as you see it here- I just wanted to make sure that the user reads it :grinning: